src/Faker/Provider/uk_UA/PhoneNumber.php

```diff
@@ -8,22 +8,28 @@ class PhoneNumber extends \Faker\Provider\PhoneNumber
 
         // International format (mobile)
         '+38050#######',
-        '+38096#######',
-        '+38067#######',
-        '+38093#######',
         '+38063#######',
-        '+38099#######',
+        '+38066#######',
```

Comment:
I have no way to tell which is right, your version or the previous one. Please provide a link to a public page explaining the phone number formats for future reference.